about/goals.qmd

```diff
@@ -0,0 +1,27 @@
+---
+title: Goals
+---
+
+The Seedcase Project is more than software, it is also about education,
+training, and the code itself.
+
+### Develop open-source software for health data
+
+Software that is open-source and targeted to diverse users, like health
+research organizations and small-to-medium sized companies.
+
+### Create training material and run workshops
+
+Specifically training on technical skills such as software and data
+engineering, while following open scientific practices.
+
+### Build and nurture an inclusive community
+
+A community that is open and inclusive that shares knowledge, learnings,
+and better practices in research data engineering and software
+engineering.
+
+### Lead by example
```

Comment:
```suggestion
## Develop open-source software for health data

Software that is open-source and targeted to diverse users, like health
research organizations and small-to-medium sized companies.

## Create training material and run workshops

Specifically training on technical skills such as software and data
engineering, while following open scientific practices.

## Build and nurture an inclusive community

A community that is open and inclusive that shares knowledge, learnings,
and better practices in research data engineering and software
engineering.

## Lead by example
```

Generally not good to jump in hierarchy (title which is header 1, to header 3) and rather to keep a consistent jump (title to header 2)

Reply:
Ah, yes, that was unintentional!